extensions/3DTILES_layers/0.0.0/schema/3DTILES_layers_implicit_tiling_subtree_EXT.schema.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
@ErixenCruz @lilleyse I think we need to solidify our naming conventions. My understanding of this is that this extension is called `3DTILES_layers_implicit_tiling` which extends `3DTILES_layers` which extends core 3D Tiles,  is this correct?

If this is indeed our first extension of an extension, should the root file be `3DTILES_layers.3DTILES_layers_implicit_tiling.schema.json` or just `3DTILES_layers_implicit_tiling.schema.json`? I don't believe we have a precident

Reply:
The mental model I've been thinking is that this is still `3DTILES_layers` but as applied to tilesets using implicit tiling. It wouldn't be a separate `3DTILES_layers_implicit_tiling` extension.